lib/Doctrine/Common/DataFixtures/Purger/ORMPurgerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Common\DataFixtures\Purger;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+/**
+ * ORMPurgerInterface
+ */
+interface ORMPurgerInterface
```

Comment:
I believe this should extend `PurgerInterface`, as `AbstractExecutor` expects the `purger` to be an instance of `PurgerInterface`, which is not guaranteed with the changes as is: https://github.com/doctrine/data-fixtures/blob/265b209150ce6bfa22c3825bfcd512f5b5c32d3b/lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php#L26.

```suggestion
interface ORMPurgerInterface extends PurgerInterface
```

Reply:
That's right, thanks, forgot to add it ^^